core/src/test/java/org/apache/iceberg/rest/responses/TestOAuthTokenResponse.java

```diff
@@ -67,24 +67,6 @@ public String serialize(OAuthTokenResponse response) throws JsonProcessingExcept
     return OAuth2Util.tokenResponseToJson(response);
   }
 
-  @Override
-  public void testHasOnlyKnownFields() {
-    Set<String> fieldsFromSpec = Sets.newHashSet();
-    Collections.addAll(fieldsFromSpec, allFieldsFromSpec());
-    try {
-      JsonNode node = JsonUtil.mapper().readValue(serialize(createExampleInstance()), JsonNode.class);
-      for (String field : fieldsFromSpec) {
-        Assert.assertTrue("Should have field: " + field, node.has(field));
-      }
-
-      for (String field : ((Iterable<? extends String>) node::fieldNames)) {
-        Assert.assertTrue("Should not have field: " + field, fieldsFromSpec.contains(field));
-      }
-    } catch (JsonProcessingException e) {
-      throw new RuntimeException(e);
-    }
-  }
```

Comment:
This is now the version of the test that exists in the base class `RequestResponseTestBase`, and thus does not need to be overridden anymore.